### PR TITLE
Add Crowdin contributors to list of contributors

### PR DESCRIPTION
**Pull request by vantezzen**
Ferdi tries to display all people who helped build the application on https://github.com/getferdi/ferdi#contributors. Unfortunately, CrowdIn doesn't natively support adding the list of contributors we have there to the all-contributors list.

This PR adds those contributors and also adds a simple script with which we can update the list in the future.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@
     - [Start development app](#start-development-app)
     - [Packaging](#packaging)
     - [Release](#release)
-  - [Contributors ✨](#contributors-)
+  - [Contributors ✨](#contributors-%e2%9c%a8)
```
Comment (kytwb) [inline README.md]: Regression here?

File: README.md [hunk 2]
```diff
@@ -82,6 +82,11 @@ Ferdi has three seperate AUR packages you can use:
 - **[ferdi-bin](https://aur.archlinux.org/packages/ferdi-bin/)**: Uses the latest Fedora release and extracts it to Arch
 - **[ferdi-git](https://aur.archlinux.org/packages/ferdi-git/)**: Uses your system electron version to run the latest commit from the develop branch
 
+We suggest using the `ferdi` package first as it will work best on most systems.
```
Comment (kytwb) [inline README.md]: Not related to contributors, are those changed volontary?

File: src/scripts/add-crowdin-contributors.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+      options: {
+        contributors: info.contributors,
+      },
+    });
+  }
+
+  fs.writeJSON(infoPath, info, {
+    spaces: 2,
+  });
+})();
```
Comment (kytwb) [inline src/scripts/add-crowdin-contributors.js]: Nice! 💪

**Comment by vantezzen** [comment]
Looks like my changes in https://github.com/getferdi/ferdi/tree/refactor/installation-guide somehow merged into the branch? Weird, but I removed those changes now.

**Comment by vantezzen** [comment]
Sorry, needed to resolve merge conflicts - there are no other changes.

**Comment by kytwb** [comment]
@vantezzen no worries! 🙏

**Pull request merged**